cmd/influx/remote.go

```diff
@@ -102,14 +102,45 @@ func newRemoteDeleteCmd() cli.Command {
 }
 
 func newRemoteListCmd() cli.Command {
+	var params remote.ListParams
 	return cli.Command{
 		Name:    "list",
 		Usage:   "List all remote connections",
 		Aliases: []string{"find", "ls"},
 		Before:  middleware.WithBeforeFns(withCli(), withApi(true), middleware.NoArgs),
-		Flags:   commonFlags(),
-		Action: func(ctx *cli.Context) {
-			fmt.Println("remote list command was called")
+		Flags: append(
+			commonFlags(),
+			&cli.StringFlag{
+				Name:        "name, n",
+				Usage:       "Name filter for remote connections list",
+				Destination: &params.Name,
+			},
+			&cli.StringFlag{
+				Name:        "org-id",
+				Usage:       "Local org ID",
+				EnvVar:      "INFLUX_ORG_ID",
+				Destination: &params.OrgID,
+			},
+			&cli.StringFlag{
+				Name:        "org, o",
+				Usage:       "Local org name",
+				EnvVar:      "INFLUX_ORG",
+				Destination: &params.OrgName,
+			},
+			&cli.StringFlag{
+				Name:        "remote-url",
+				Usage:       "Remote URL filter for remote connections list",
+				Destination: &params.RemoteURL,
+			},
+		),
+		Action: func(ctx *cli.Context) error {
+			client := remote.Client{
+				CLI:                  getCLI(ctx),
+				RemoteConnectionsApi: getAPI(ctx).RemoteConnectionsApi,
+				OrganizationsApi:     getAPI(ctx).OrganizationsApi,
```

Comment:
`getAPI` does some nontrivial setup work, could you pull it into a local var so we don't have to call it twice?